Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -485,6 +485,70 @@ def condition_(self, fragment, stepList, key, hasHarvest):
     '--procModifiers': 'trackingMkFitCommon,trackingMkFitInitialStep'
 }
 
+# LST on CPU, initialStep+highPtTripletStep-only tracking-only
+class UpgradeWorkflow_lstOnCPUIters01TrackingOnly(UpgradeWorkflowTracking):
+    def setup__(self, step, stepName, stepDict, k, properties):
+        if 'Reco' in step: stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
+        elif 'HARVEST' in step: stepDict[stepName][k] = merge([{'-s': 'HARVESTING:@trackingOnlyValidation+@trackingOnlyDQM'}, stepDict[step][k]])
+        elif 'ALCA' in step: stepDict[stepName][k] = None
+    def condition_(self, fragment, stepList, key, hasHarvest):
+        return ('2026' in key)
+upgradeWFs['lstOnCPUIters01TrackingOnly'] = UpgradeWorkflow_lstOnCPUIters01TrackingOnly(
+    steps = [
+        'Reco',
+        'RecoFakeHLT',
+        'HARVEST',
+        'HARVESTFakeHLT',
+        'RecoGlobal',
+        'HARVESTGlobal',
+        'RecoNano',
+        'RecoNanoFakeHLT',
+        'HARVESTNano',
+        'HARVESTNanoFakeHLT',
+        # Add ALCA steps explicitly, so that they can be properly removed
+        'ALCA',
+        'ALCAPhase2'
+    ],
+    PU = [],
+    suffix = '_lstOnCPUIters01TrackingOnly',
+    offset = 0.341,
```

Comment:
I'd continue indexing wrt the above wf; so, `.703`

Reply:
BTW, why so many patterns in the `steps` compared to the mkFit case above?
Are we missing something in the mkfit?

Reply:
> I'd continue indexing wrt the above wf; so, .703

 Will update accordingly.

> BTW, why so many patterns in the steps compared to the mkFit case above?
> Are we missing something in the mkfit?

No but I realized the following: unless a step is explicitly defined, one cannot modify it through `setup__`. In the LST case, I need to modify the `HARVESTING` step, so that I can make it `trackingOnly`, and I need to disable the `ALCA` step, hence they were added. Of course, there may be a way to modify them without explicitly adding them (which would naively make sense to me), but I couldn't make it work.